server/modules/instantiation/jobs.js

```diff
@@ -88,6 +88,13 @@ module.exports = function (config, portProvider, interpolationHelper, buildInsta
         }
     }
 
+    class RunAfterBuildTaskJob extends ComponentInstanceJob {
```

Comment:
actually this one has been already reported in https://github.com/xsolve-pl/xsolve-feat/issues/13